sxg_rs/src/lib.rs

```diff
@@ -318,6 +322,12 @@ impl SxgWorker {
     }
 }
 
+impl From<Config> for SxgWorker {
+    fn from(config: Config) -> Self {
+        Self { config }
+    }
+}
+
```

Comment:
It is possible that `SxgWorker` will have more member variables other than `config` itself in the future, so I think we should not add this.

Reply:
Perhaps I can change it to a new constructor like `new_with_config(config: Config)`?

Reply:
Yes.